### PR TITLE
fix #1578 better checkin question rotation

### LINKED ISSUE #1578: faster Social Check-In prompt rotation
## Issue - Enhancement

When team members are on several teams (in our case, FOUR) that tend to meet on the same day/s, often with some of the same mix of people, it would be nice if the **social check-in prompt randomized hourly so that each meeting gets a new question.**

### Acceptance criteria:
Implemented in:
- [x] retro meeting
- [x] existing action meeting

### PR DESCRIPTION
**Pull request by mattkrick**
TEST
- [ ] create 3 teams
- [ ] start an action meeting on each team
- [ ] verify at least 1 has a different question (there's ~5% chance that you'll have a duplicate)
- [ ] take all meetings to completion
- [ ] start 3 meetings again
- [ ] verify a different question for all 3

### CODE REVIEW

**Comment by ackernaut** [comment]
Going to review this one now

**Comment by ackernaut** [comment]
#### Test No. 1 (in progress)
- [ ] create 3 teams
- [ ] start an action meeting on each team
- [ ] verify at least 1 has a different question (there's ~5% chance that you'll have a duplicate)
- [ ] take all meetings to completion
- [ ] start 3 meetings again
- [ ] verify a different question for all 3

File: src/server/graphql/mutations/startMeeting.js [hunk 1]
```diff
@@ -39,11 +38,15 @@ export default {
...
+    const meetingCount = await r.table('Meeting')
+      .getAll(teamId, {index: 'teamId'})
+      .count()
+      .default(0)
+      .add(1);
 
     const updatedTeam = {
-      checkInGreeting: makeCheckinGreeting(week, teamId),
-      checkInQuestion: convertToTaskContent(makeCheckinQuestion(week, teamId)),
+      checkInGreeting: makeCheckinGreeting(meetingCount, teamId),
```
[the remark this answers is not in the document]
Comment (mattkrick) [reply src/server/graphql/mutations/startMeeting.js]: almost! the first character of the teamId picks which greeting we start at. Then, we add the # of meetings we've had to that. so, greetings will be unique as long as they have IDs that start with a different character OR they are on a different meeting number.

[the remark this answers is not in the document]
Comment (ackernaut) [reply src/server/graphql/mutations/startMeeting.js]: nice! I need to brush up on reading the methods on r.table

**Pull request merged**